docs/Contributors_Guide/directory_tree.rst

```diff
@@ -58,6 +58,9 @@ Sample Structure::
     __init__.py
     logger.py
     exceptions.py
+
+.. COMMENT: Any reason this directory tree doesn't use the same structure 
+   as the one above with "├──" and "└──"?
```

Comment:
The one above was likely generated with a linux tool `tree`, but when I created this one, I didn't use `tree`. I could try to get that change in if it's important.

Reply:
@christinaholtNOAA I didn't know there was a tool for that! I already made a formatted directory tree but then removed it just in case there was a reason you all had decided against. Just added it back in and removed the comment. Let me know if it's okay or if it needs any changes.